frontend/tests/features/login.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Admin login
+  As an admin
+  I want to login
+  So that I can view home page
+
+  Background:
+    Given admin has navigated to login page
+
+  Scenario: Login with valid credentials
+    When admin login with following credentials
+      | username | password |
+      | admin    | admin    |
+    Then admin should be navigated to home page
```

Comment:
We can add a step that checks the response code before this step.
Like Then the HTTP status code should be "code" for API test.